sdcm/sct_config.py

```diff
@@ -1622,6 +1623,9 @@ class SCTConfiguration(dict):
     def __init__(self):
         # pylint: disable=too-many-locals,too-many-branches,too-many-statements
         super().__init__()
+        self.scylla_version = None
+        self.is_enterprise = False
```

Comment:
I don't see these new attrs to be used in commits of this PR.

Reply:
I would be using them for enabling KMS by default on enterprise

Reply:
it would be used for https://github.com/scylladb/scylla-cluster-tests/pull/6815